sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -246,7 +247,27 @@ case class LoadDataCommand(
     val loadPath =
       if (isLocal) {
         val uri = Utils.resolveURI(path)
-        if (!new File(uri.getPath()).exists()) {
+        val filePath = uri.getPath()
+        val exists = if (filePath.contains("*")) {
+          val splitPath = filePath.split(File.separator)
+          val filePattern = splitPath.last
+          val dir = splitPath.dropRight(1).mkString(File.separator)
```

Comment:
I usually write `.init` to mean "all but the last element"
Hm, if you're using the local file system here, can you use the JDK `java.nio.file.Path` API to do the parsing of elements?